apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -28,51 +29,67 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 	);
 
 	return (
-		<section className={styles["signup-container"]}>
-			<div className={styles["logo-container"]}>
-				<div
-					className={getValidClassNames(
-						styles["circle"],
-						styles["circle-pink"],
-					)}
-				/>
-				<h1>LOGO</h1>
-			</div>
-			<h3 className={styles["title-form"]}>CREATE AN ACCOUNT</h3>
-			<span className={styles["auth-info"]}>
-				Already have an account? Go to
-				<Link to={AppRoute.SIGN_IN}> Sign in</Link>
-			</span>
-			<form onSubmit={handleFormSubmit}>
-				<div className={styles["input-groups"]}>
-					<Input
-						control={control}
-						errors={errors}
-						label="Name"
-						name="name"
-						placeholder="name"
-						type="text"
-					/>
-					<Input
-						control={control}
-						errors={errors}
-						label="Email"
-						name="email"
-						placeholder="email"
-						type="email"
-					/>
-					<Input
-						control={control}
-						errors={errors}
-						label="Password"
-						name="password"
-						placeholder="password"
-						type="password"
-					/>
+		<>
+			<form className={styles["form"]} onSubmit={handleFormSubmit}>
+				<div className={styles["form-header"]}>
+					<div className={styles["form-header__logo-container"]}>
+						<div className={styles["form-header__logo"]} />
+						<span className={styles["form-header__logo-text"]}>logo</span>
+					</div>
+
+					<h1 className={styles["form-header__text"]}>create an account</h1>
+					<span className={styles["form-header__sub-text"]}>
+						Already have an account? Go to{" "}
+						<Link to={AppRoute.SIGN_IN}>Sign in</Link>
+					</span>
 				</div>
+
```

Comment:
Not fixed. Should be outside of `<form>` tag. Same for SignInForm

Reply:
So only input fields should be inside of form?

Reply:
That's right. The logo and links are not a part of a form